src/@types/ROAM/index.d.ts

```diff
@@ -0,0 +1,19 @@
+declare module 'roam' {
+  import { GenericObject } from '../../utilTypes'
```

Comment:
`utilTypes` was removed and `GenericObject` was renamed to `Index`. Not sure how this is compiling.

Reply:
You're right. I found out that it's being ignored by `eslint` based on its current configuration.

Reply:
Yes, thanks!

Reply:
@raineorshine  Do you think we can do this in a different PR or maybe towards the end of this PR because there are other `.d.ts` files as well that might need to be handled?

Reply:
Yes, let's do a separate PR. I'd like to know what your overall plan is before starting on it.